pages/gallery.html

```diff
@@ -4,9 +4,70 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Gallery - All About Coffee</title>
+  <link href="../styles/style.css" rel="stylesheet">
+  <link href="../styles/gallery.css" rel="stylesheet">
 </head>
 <body>
+  <header>
+    <nav>
+      <ul class="site-menu">
+        <li><a class="home-to-home" href="../index.html"><button>HOME</button></a></li>
+        <li><a class="home-to-gallery" href="#"><button>GALLERY</button></a></li>
+        <li><a class="home-to-facts" href="./facts.html"><button>FUN FACTS</button></a></li>
+      </ul>
+    </nav>
+    <h1>Gallery</h1>
+  </header>
+
+  <main>
+    <section>
+      <h2>Drip Coffee</h2>
+      <img src="../images/drip-coffee.jpg" alt="drip coffee" style="height: auto; width: 400px;">
```

Comment:
Inline styling is not common practice. Move the styling here to an external stylesheet.